.goreleaser.yml

```diff
@@ -0,0 +1,70 @@
+before:
+  hooks:
+    - go mod download
+    - go mod verify
+builds:
+  - main: ./cmd/mattermost-mattermod
+    id: "mattermod"
+    binary: bin/mattermod
+    goos:
+      - linux
+    goarch:
+      - amd64
+    flags:
+      - -mod=readonly
+      - -trimpath
+    ldflags:
+      - -X github.com/mattermost/mattermost-mattermod/version.version=v{{.Version}}
+      - -X github.com/mattermost/mattermost-mattermod/version.commitHash={{.FullCommit}}
+      - -X github.com/mattermost/mattermost-mattermod/version.buildDate={{.Date}}
+    env:
+      - CGO_ENABLED=0
+  - main: ./cmd/jobserver
+    id: "jobserver"
+    binary: bin/jobserver
+    goos:
+      - linux
+    goarch:
+      - amd64
+    flags:
+      - -mod=readonly
+      - -trimpath
+    ldflags:
+      - -X github.com/mattermost/mattermost-mattermod/version.version=v{{.Version}}
+      - -X github.com/mattermost/mattermost-mattermod/version.commitHash={{.FullCommit}}
+      - -X github.com/mattermost/mattermost-mattermod/version.buildDate={{.Date}}
+    env:
+      - CGO_ENABLED=0
+  - main: ./cmd/migrator
+    id: "migrator"
+    binary: bin/migrator
+    goos:
+      - linux
```

Comment:
This is essentially what's needed and it's a tool for operators so we need to publish all available os (`darwin`, `windows`, `linux` and maybe just throwing `freebsd` out there too.)